### ISSUE TITLE
Why binary file version not matched release version number?

### ISSUE THREAD

**Issue by pedoc** [nushell/nushell#5631]
### Describe the bug

![image](https://user-images.githubusercontent.com/6604230/170172642-b4389bb6-8608-4e80-b16c-ddb6266d201c.png)
![image](https://user-images.githubusercontent.com/6604230/170172669-52a7e22c-aab3-40d0-85d1-48fd62dbdf2b.png)

Note that the file version number is defined here
``https://github.com/nushell/nushell/blob/main/assets/nushell.rc``

Is this a design or a bug?

### How to reproduce

1. install latest version nushell 
2. print nu version
3. get version from windows explorer

### Expected behavior

The file version is the same as the release version number

**Comment by rgwood** [comment]
Good timing! I'm just about to merge a PR that will fix this: https://github.com/nushell/nushell/pull/5630

**Comment by rgwood** [comment]
> Is this a design or a bug?

It's a bug. Previously we needed to manually update the file version in `nushell.rc` and it looks like that was missed in Nushell v0.63. I've just merged a PR that will automatically keep the file version in sync with the real Nushell version, and this is what nu.exe now looks like after a build:

<img width="283" alt="image" src="https://user-images.githubusercontent.com/26268125/170174024-69f02f86-82df-4ec8-b7b3-6b3bd34a9681.png">

Thanks for the bug report!
